<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Helvetica</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>CBNU Grading Management System</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>561</width>
     <height>251</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>이름</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>연도</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>학기</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>과목</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>이수구분</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>학점</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>등급</string>
    </property>
   </column>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>601</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>15</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_logout">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>430</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>로그아웃</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_download">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>40</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>다운로드</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_refresh">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>430</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>새로고침</string>
   </property>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>tableWidget</zorder>
  <zorder>pushButton_logout</zorder>
  <zorder>pushButton_download</zorder>
  <zorder>pushButton_refresh</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
